README.md

```diff
@@ -73,3 +73,13 @@ Unless you explicitly state otherwise, any contribution intentionally submitted
 for inclusion in the work by you, as defined in the Apache-2.0 license, shall be
 dual licensed as above, without any additional terms or conditions.
 </sub>
+
+## Third-party middleware
+
+The following third-party middleware use `request-middleware`:
+
+- https://github.com/oxidecomputer/reqwest-conditional-middleware
+- https://github.com/06chaynes/http-cache
+- https://gitlab.com/famedly/company/backend/libraries/reqwest-cache
+- https://github.com/neoeinstein/aliri/tree/main/aliri_reqwest
+- https://crates.io/crates/http-signature-normalization-reqwest (not free software)
```

Comment:
Probably worth adding a short description of what each of them does, before the link itself?